ConsumerApi/appsettings.override.json

```
@@ -187,7 +187,7 @@
       {
         "Name": "Console",
         "Args": {
-          "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss}|{Level} => CorrelationID:{CorrelationID} => RequestId:{RequestId} => RequestPath:{RequestPath}{NewLine}    {SourceContext}{NewLine}    {Message}{NewLine}{Exception}"
+          "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss}|{Level} => CorrelationID:{CorrelationId} => RequestId:{RequestId} => RequestPath:{RequestPath}{NewLine}    {SourceContext}{NewLine}    {Message}{NewLine}{Exception}"
```

I know that this wasn't part of the task, but it's bothering me for quite some time now. :)

1. We should be consistent with the naming. Currently We have CorrelationID and RequestId (watch the casing of the id)
2. I don't like the arrows serving as separators. Please use "-" instead
3. Please add milliseconds to the timestamp

```suggestion
          "outputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss}|{Level} - CorrelationId:{CorrelationId} - RequestId:{RequestId} - RequestPath:{RequestPath}{NewLine}    {SourceContext}{NewLine}    {Message}{NewLine}{Exception}"
```